client/components/helpCentre/liveChat/LiveChat.tsx

```diff
@@ -14,6 +14,164 @@ import { liveChatCss } from './liveChatCssOverrides';
 
 let areAgentsAvailable = false;
 
+// Enhanced Chat types
+declare global {
+	interface Window {
+		embeddedservice_bootstrap?: {
+			settings: {
+				language: string;
+			};
+			init: (
+				orgId: string,
+				serviceName: string, // The API Name of your deployment
+				siteUrl: string,
+				options?: object,
+			) => Promise<void>;
+			utilAPI: {
+				launchChat: () => void;
+				hideChatButton: () => void;
+			};
+			prechatAPI: {
+				setHiddenPrechatFields: (
+					fields: Record<string, string>,
+				) => void;
+				setVisiblePrechatFields: (
+					fields: Record<string, object>,
+				) => void;
+			};
+			userVerificationAPI?: {
+				setIdentityToken: (token: object) => Promise<void>;
+			};
+		};
+	}
+}
+
+// Configuration for Enhanced Chat environments
+const MIAW_CONFIG = {
+	PROD: {
+		//not ready for production yet
+		URL: 'https://example.com/ESWDeployment',
+		ORG_ID: '00D200000001234',
+		DEPLOYMENT_NAME: 'Example_Deployment_Name',
+	},
+	CODE: {
+		URL: 'https://gnmtouchpoint--dev1.sandbox.my.site.com/ESWEnhancedLiveChat1765817908909',
+		ORG_ID: '00D9E0000004jvh',
+		DEPLOYMENT_NAME: 'Enhanced_Live_Chat',
+	},
+};
+
+const getChatConfig = () => {
+	return window.guardian.domain === 'theguardian.com'
+		? MIAW_CONFIG.PROD
+		: MIAW_CONFIG.CODE;
+};
+
+// Initialize function
+const initEnhancedChat = () => {
+	return new Promise((resolve, reject) => {
+		const config = getChatConfig();
+
+		console.log('Starting init function');
+
+		// If API is already fully ready, return immediately
+		if (window.embeddedservice_bootstrap?.utilAPI) {
+			resolve(true);
+			return;
+		}
+
+		// Setup the "Ready" Listener
+		// This event fires when the chat is fully loaded and APIs are usable.
+		const onReadyHandler = () => {
+			resolve(true);
+
+			window.removeEventListener(
+				'onEmbeddedMessagingReady',
+				onReadyHandler,
+			);
+		};
+		window.addEventListener('onEmbeddedMessagingReady', onReadyHandler);
+
+		// Load Script if missing
+		if (!window.embeddedservice_bootstrap) {
+			const script = document.createElement('script');
+			script.src = `${config.URL}/assets/js/bootstrap.min.js`;
+			script.onload = async () => {
+				try {
+					if (!window.embeddedservice_bootstrap) {
+						//TODO: Is this the correct way to work around "'window.embeddedservice_bootstrap' is possibly 'undefined'" ?
+						throw new Error(
+							'Embedded Service Bootstrap failed to load.',
+						);
+					}
+
+					// Ensure settings exist
+					window.embeddedservice_bootstrap.settings =
+						window.embeddedservice_bootstrap.settings || {};
+					window.embeddedservice_bootstrap.settings.language =
+						'en_US';
```

Comment:
May need to return to this in future - it doesn't seem to work with any other value despite configuring it in Salesforce to use English (United Kingdom) (e.g. `en_GB`).

Also it doesn't work if `language` isn't explicitly set, there's no default.